GMnetENGINE.gmx/objects/obj_upnp.object.gmx

```diff
@@ -202,16 +201,19 @@ with (global.htme_object)
             <kind>1</kind>
             <string>/// Get local IP
 getlocalip=true;
-with global.htme_object
+
+var buffer=global.udphp_server_buffer;
+var socketOrServer=global.udphp_server_udp;
+var port=global.udphp_upnp_port;
+
+if socketOrServer&gt;-1
 {
```

Comment:
There is a &gt; here instead of an actual >.

Is that correct? It might just be a new formatting thing of the new Game Maker versions, couldn't test it yet.